doc.go

```diff
@@ -195,6 +195,14 @@ Note: The logger may be changed in the future for better logging. Currently if
 the applications use the same parameters as glog, you cannot collect both
 application and driver logs at the same time.
 
+Query request ID
+
+Specific query request ID can be set in the context and will be passed through
```

Comment:
Trivial: Start with "A", i.e. "A specific query request ID can be ...".

Reply:
Yes I can fix, anything else? Thanks